src/textual/scrollbar.py

```diff
@@ -205,18 +205,18 @@ def __rich_repr__(self) -> rich.repr.Result:
         yield "window_size", self.window_size
         yield "position", self.position
 
-    def render(self) -> RenderableType:
-        styles = self.parent.styles
+    def render(self, style: Style) -> RenderableType:
+        style = self.parent.styles
```

Comment:
This is overriding the incoming `style` object. It should perhaps be renamed `styles`.

Reply:
Very good spot